helm/kagent/templates/deployment.yaml

```diff
@@ -50,6 +50,10 @@ spec:
           {{- with .Values.controller.env }}
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.ports.controller.targetPort }}
```

Comment:
@EItanya you fine with exposing this (mostly for testing purposes)?

Reply:
Yea I don't see why not, seems good to have anyway